bin/pygrb/pycbc_grb_inj_finder

```diff
@@ -284,7 +283,7 @@ with tqdm.tqdm(args.inj_files, desc="Finding injections",
                 found.append(i)
                 eid = l if r - l == 1 else event_id[l + snr[l:r].argmax()]
                 for col in triggers:
-                    trigs[col].append(triggers[col][i])
+                    trigs[col].append(triggers[col][i - len(allinjections or [])])
```

Comment:
I think this should actually be:

```suggestion
                    trigs[col].append(triggers[col][eid])
```

but I'm not sure. I don't have a test run that I can use to confirm, so if someone does, please point me in the right direction.